src/exceptions/InvalidException.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+class InvalidException extends InvalidArgumentException implements IStatusException
```

Comment:
Maybe it would be more clear if the exception name was similar to the response type?
```suggestion
class UnprocessableEntityException extends InvalidArgumentException implements IStatusException
```